relay-event-derive/src/lib.rs

```diff
@@ -90,7 +90,13 @@ fn derive_process_value(mut s: synstructure::Structure<'_>) -> TokenStream {
 
         let mut body = TokenStream::new();
         for (index, bi) in variant.bindings().iter().enumerate() {
-            let field_attrs = parse_field_attributes(index, bi.ast(), &mut is_tuple_struct);
+            let mut field_attrs = parse_field_attributes(index, bi.ast(), &mut is_tuple_struct);
+            // In case we have `fake_trim` set on the type, we want to override this field attribute
+            // manually.
+            if type_attrs.fake_trim {
```

Comment:
It would be nice not to use `field_attrs` and instead create a new type but it might make it a much bigger PR and maybe confuse whoever uses the `Processor` interface.